lms/lib/comment_client/user.py

```diff
@@ -113,7 +113,12 @@ def subscribed_threads(self, query_params={}):
             metric_tags=self._metric_tags,
             paged_results=True
         )
-        return response.get('collection', []), response.get('page', 1), response.get('num_pages', 1)
+        return (
+            response.get('collection', []),
+            response.get('page', 1),
+            response.get('num_pages', 1),
+            response.get('thread_count'),
+        )
```

Comment:
I agree that we should aim to (eventually) have one discussion api for both mobile and web.  As far as the active_threads response goes, I don't think there would be a problem with adding a field (if the change is that simple), but @nasthagiri might be able to suggest someone who would have deeper knowledge of how that's being used on the web side.